transform/app/tasks/utils/send.py

```diff
@@ -3,15 +3,66 @@
 import logging
 from typing import Optional
 
+import boto3
+from pyspark.sql import DataFrame
+
 from app.services.celery.task import CeleryTaskStatus
+from app.services.celery.task_statuses import FAILURE, SUCCESS
+from app.services.s3.send import send_spark_df as send_to_s3
+from app.services.solr.send import send_spark_df as send_to_solr
+from app.settings import settings
+from app.tasks.utils.s3_paths import extract_after_bucket
 from app.worker import celery
 
 logger = logging.getLogger(__name__)
 
 
 @celery.task(name="send_data")
-def send_data(prev_task_status: Optional[CeleryTaskStatus], req_body: dict) -> dict:
+def send_data(
```

Comment:
this function is /dump specific it would be nice to write it smaller and more generic to handle also sending data for /full and /batch endpoints (they do not have instances).